Appraisals

```diff
@@ -11,4 +11,5 @@ end
 appraise "latest" do
   gem "rails"
   gem "sidekiq"
+  gem "sprockets-rails"
```

Comment:
I doubt you need this. If you do, please include the errors that happen if you don't so I can look into it further.

Reply:
```
Failure/Error: require "sprockets/railtie"

LoadError:
  cannot load such file -- sprockets/railtie
# ./spec/dummy/config/application.rb:5:in `require'
# ./spec/dummy/config/application.rb:5:in `<top (required)>'
# ./spec/dummy/config/environment.rb:2:in `require_relative'
# ./spec/dummy/config/environment.rb:2:in `<top (required)>'
# ./spec/rails_helper.rb:4:in `require'
# ./spec/rails_helper.rb:4:in `<top (required)>'
# ./spec/simple_scheduler/at_spec.rb:1:in `require'
# ./spec/simple_scheduler/at_spec.rb:1:in `<top (required)>'

An error occurred while loading ./spec/simple_scheduler/future_job_spec.rb. - Did you mean?
                    rspec ./spec/simple_scheduler/scheduler_job_spec.rb
                    rspec ./spec/simple_scheduler/at_spec.rb
                    rspec ./spec/simple_scheduler/task_spec.rb
```